obelisk_ws/src/obelisk_msgs/obelisk_control_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
In general its hard to say exactly what we should set this too. 3.0 and greater are very different, so we want at least that (which it is). That being said, I think we can get away with higher than 3.5. I have been using min 3.16 in my projects recently, but there isn't necessarily going to be any notable difference. I think we should bump this to 3.16 though, as that is what we have in the pixi.toml.

Reply:
done.